File: tests/test_homepage.js
const {Builder} = require("selenium-webdriver");
const firefox = require('selenium-webdriver/firefox');

const options = new firefox.Options();
options.setBinary("./geckodriver");

async function testHomepage() {
    let driver = await new Builder().forBrowser("firefox").setFirefoxOptions(options).build();

    await driver.get("http://127.0.0.1:8080/");

    const header = driver.findElement(By.xpath("//*[text()='Krishanteringssystem']"));

    console.log(header)

    await driver.quit();
};

testHomepage();

File: tests/test_homepage.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from web_test_base import WebTestBase
from selenium.common.exceptions import NoSuchElementException

class TestHomepage(WebTestBase):

    def test_homepage(self):
        driver = self.driver
        driver.get(self.WEBSITE_URL)

        try:
            header = driver.find_element(By.XPATH, "//*[contains(text(), 'Krishanteringssystem')]")
        except NoSuchElementException:
            self.fail("Could not find element with text: 'Krishanteringssystem'")